laletterbuilder/tests/test_schema.py

```diff
@@ -474,3 +472,51 @@ def test_it_returns_none_if_no_letters_exist_for_user(self):
     def test_it_returns_letter_if_one_exists_for_user(self):
         self.graphql_client.request.user = self.letter.user
         assert self.execute()["trackingNumber"] == "abcd"
+
+
+class TestLaLetterBuilderSendOptionsMutation:
+    @pytest.fixture(autouse=True)
+    def setup_fixture(self, graphql_client, db):
+        self.user = UserFactory(email="boop2@jones.net")
+        graphql_client.request.user = self.user
+        self.graphql_client = graphql_client
+
+    def execute(self, input):
+        return self.graphql_client.execute(
+            """
+            mutation MyMutation($input: LaLetterBuilderSendOptionsInput!) {
+                output: laLetterBuilderSendOptions(input: $input) {
+                    errors {
+                        field
+                        messages
+                    }
+                    session {
+                        habitabilityLatestLetter {
+                            mailChoice
+                        }
+                        landlordDetails {
+                            email
+                        }
+                    }
+                }
+            }
+            """,
+            variables={"input": input},
+        )["data"]["output"]
+
+    @pytest.mark.django_db
+    def test_it_saves_landlord_email_and_mail_choice(self, db):
```

Comment:
You may also want to test that any existing landlord details aren't harmed/altered/erased when saving this new email field.